values-nl/strings.xml

```diff
@@ -2,23 +2,23 @@
+            <string name="appintro_root_title">Rootrechten</string>
+            <string name="appintro_label_android">Android-versie</string>
+            <string name="appintro_label_architecture">CPU-architectuur</string>
             <string name="appintro_label_variant">Pakketvariant</string>
             <string name="label_tos">Gebruiksvoorwaarden</string>
         <!--DESCRIPTIONS-->
-            <string name="appintro_storageperm_explain">Om een Open GApps-pakket te kunnen downloaden, dient u de app toestemming te geven tot uw Bestandssysteem</string>
-            <string name="slide_arch_description">Selecteer a.u.b. de architectuur die u wilt gebruiken voor uw Open GApps pakket</string>
-            <string name="slide_android_description">Selecteer a.u.b. de Android versie die u wilt gebruiken voor uw Open GApps pakket</string>
-            <string name="slide_variant_description">Selecteer a.u.b. de pakket variant die u wilt gebruiken voor uw Open GApps pakket</string>
-            <string name="appintro_root_explanation">De app biedt een geïntegreerde installatie-modus om direct een gedownload pakket te installeren. Deze installatie-modus vereist <i>root rechten</i>. Als u dit wenst te gebruiken, dient u ons root rechten te verlenen. \nDeze stap is optioneel.</string>
-            <string name="appintro_terms_explain">De kant-en-klaar pakketten van OpenGApps.org worden enkel aangeboden als een gunst en zonder enige vorm van garantie, op voorwaarde dat deze enkel gebruikt mogen worden voor persoonlijk gebruik, en niet verder verspreid worden aan het publiek dan via OpenGApps.org. \n \n Let op dat Open GApps u geen licentie verstrekt voor Google’s APKs in het pakket. De Open GApps pakketten bieden slechts een gemakkelijke methode om APKs op je apparaat te laden. Het is uw eigen verantwoordelijkheid om de benodigde toestemmingen door bijvoorbeeld een OHA-gelicenseerd apparaat met voorgeïnstalleerde Google Apps te kopen en/of de applicaties van Google’s Play Store te betrekken.</string>
+            <string name="appintro_storageperm_explain">Om een Open GApps-pakket te kunnen downloaden, dient u de app toestemming te geven tot uw bestandssysteem</string>
+            <string name="slide_arch_description">Selecteer a.u.b. de architectuur die u wilt gebruiken voor uw Open GApps-pakket</string>
+            <string name="slide_android_description">Selecteer a.u.b. de Android-versie die u wilt gebruiken voor uw Open GApps-pakket</string>
+            <string name="slide_variant_description">Selecteer a.u.b. de pakketvariant die u wilt gebruiken voor uw Open GApps-pakket</string>
+            <string name="appintro_root_explanation">De app biedt een geïntegreerde installatiemodus om direct een gedownload pakket te installeren. Deze installatiemodus vereist <i>rootrechten</i>. Als u dit wenst te gebruiken, dient u ons rootrechten te verlenen. \nDeze stap is optioneel.</string>
```

Comment:
Ik zie in veel andere apps zulke woorden zonder koppelteken terugkomen. Persoonlijk vind ik de woorden goed leesbaar. Ik zou er geen koppeltekens tussen plaatsen.